source/reference/permissions.txt

```diff
@@ -0,0 +1,27 @@
+.. _c2c-permissions-and-roles:
+
+================
+User Permissions
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/fact-permissions-body.rst
+
+
+Self-Managed Clusters
+---------------------
+
+.. include:: /includes/table-permissions-self-hosted.rst
+
+Atlas Clusters
+--------------
+
+.. include:: /includes/table-permissions-atlas.rst
+
```

Comment:
**Suggestion**

Would it be valuable to add a "Learn More" section and link to the manual to view the granular grants of each built in role? 

Ex:

See all permission grants for the following roles:
- [ClusterManager](https://www.mongodb.com/docs/manual/reference/built-in-roles/#mongodb-authrole-clusterManager)
- [Restore](https://www.mongodb.com/docs/manual/reference/built-in-roles/#mongodb-authrole-restore)

Reply:
Done

Reply:
Nice, this is great context.

Reply:
Thanks!